pkg/controller/master/backup/restore.go

```diff
@@ -209,6 +230,38 @@ func (h *RestoreHandler) RestoreOnRemove(key string, restore *harvesterv1.Virtua
 	return nil, nil
 }
 
+// pv naming convention from externel-provisioner
+// porting from https://github.com/kubernetes-csi/external-provisioner/blob/90eae32d3a7352590500073b72b9a07f43adc881/pkg/controller/controller.go#L420-L436
+func makeVolumeName(prefix, pvcUID string, volumeNameUUIDLength int) (string, error) {
+	// create persistent name based on a volumeNamePrefix and volumeNameUUIDLength
+	// of PVC's UID
+	if len(prefix) == 0 {
+		return "", fmt.Errorf("Volume name prefix cannot be of length 0")
+	}
+	if len(pvcUID) == 0 {
+		return "", fmt.Errorf("corrupted PVC object, it is missing UID")
+	}
+	if volumeNameUUIDLength == -1 {
```

Comment:
```suggestion
	if volumeNameUUIDLength == volumeNameUUIDNoTruncate {
```

Reply:
This is ported from` external-provisioner`, I'd like to keep it as the origin, thanks.